onnxruntime/core/providers/qnn/builder/qnn_node_group/conv_activation_fusion.cc

```diff
@@ -53,11 +54,11 @@ static bool GetQScalarScaleZeroPoint(const QnnModelWrapper& qnn_model_wrapper,
   return true;
 }
 
-// Computes the floating point range (rmin, rmax) from a QuantizeLinear node's scale/zero-point.
-static bool GetQRminRmax(const QnnModelWrapper& qnn_model_wrapper,
-                         const NodeUnit& q_node_unit,
-                         /*out*/ float& rmin,
-                         /*out*/ float& rmax) {
+// Computes the floating point domain (min_, max_) from a QuantizeLinear node's scale/zero-point.
+static bool GetQDomain(const QnnModelWrapper& qnn_model_wrapper,
```

Comment:
Is it possible to have a shared implementation of the logic for this and #22580 to use?

Reply:
I can refactor the `GetQDomain` in [onnxruntime/core/optimizer/qdq_transformer/clip_quantizelinear.cc](https://github.com/microsoft/onnxruntime/pull/22580/files?w=1#diff-d263e4275790b1e6e406f2513688257815e385230bc3a22f11566f611e6778b8) in this PR and reuse it here.

Reply:
Seems only the `switch` part can be reused, which only improves very little. Lets just keep it as is if no future reuse pop up.